v2/stacks/cmd/deps.yml

```diff
@@ -41,6 +41,10 @@ services:
     extends:
       file: ${PATH_MANIFESTS}/deps/elasticsearch.yml
       service: cmdelasticsearch
+  babbageelasticsearch:
+    extends:
+      file: ${PATH_MANIFESTS}/deps/elasticsearch.yml
+      service: sitewideelasticsearch
```

Comment:
This is extending sitewidesearch (es7.10) but calling it babbageelasticsearch which I would consider as es2.4.2. Given you're not actually connecting to it in the other manifests are you sure this is required?

Reply:
I've removed this as it's no longer required.